layers/+tools/shell/funcs.el

```diff
@@ -57,28 +57,28 @@
                                        100)
                                     (window-width)))))
 
-(defmacro make-shell-pop-command (func &optional shell)
+(defmacro make-shell-pop-command (name func &optional shell)
```

Comment:
I like this change, it's a bit cleaner this way :+1:

Reply:
Thanks ✨